### PR TITLE
Rename project, reset version and changelog

### PR DESCRIPTION
**Pull request by forbesjo**
## Description
This PR updates project information from videojs-contrib-hls to http-streaming

## Requirements Checklist
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "videojs-contrib-hls",
-  "version": "5.12.2",
-  "description": "Play back HLS with video.js, even where it's not natively supported",
+  "name": "@videojs/http-streaming",
+  "version": "0.1.0",
+  "description": "Play back HLS and DASH with video.js, even where it's not natively supported",
   "main": "es5/videojs-contrib-hls.js",
```
Comment (forbesjo) [inline package.json]: What do we want the output files to be named? `videojs-http-streaming.js`?

Comment (gesinger) [reply package.json]: Sounds good to me.

Comment (mjneil) [reply package.json]: Makes sense to me

**Comment by mjneil** [comment]
maybe update README to say new project name?

**Pull request merged**